### ISSUE TITLE
Diaphora support based on RadecoIR (not for sushant and dkreuter)

### ISSUE THREAD

**Issue by XVilka** [radare/radeco#12]
Related to https://github.com/radare/radare2/issues/2742

Probably, using higher level IR for diffing makes more sense?

**Comment by radare** [comment]
Yeah probably an optimized version of radecoir can be better for diffing. Current r2 diffing engine works quite well. But it lacks an ui. I would not priorize supporting diffinhg based on radeco because it will be handy when integrated with r2. And we have to priorize the decompilation.

**Comment by XVilka** [comment]
yep, it is a very low priority and a long term goal. Just to keep this in mind
